locales/zh-CN.json

```diff
@@ -461,6 +461,7 @@
     "Standard YouTube license": "标准 YouTube 许可证",
     "Download is disabled": "已禁用下载",
     "Import YouTube playlist (.csv)": "导入 YouTube 播放列表（.csv）",
+    "Import YouTube watch history (.json)": "导入 YouTube 观看历史（.json）",
```

Comment:
Please add an entry to `en-US.json` as well (weblate gets the keys for other languages from the en-US.json file)

Reply:
Done in [81a4f29](https://github.com/iv-org/invidious/pull/4171/commits/81a4f29c735f1b067c5e53d4e49d626bc6e787e3).